src/main/java/com/minecolonies/sounds/FishermanSounds.java

```diff
@@ -0,0 +1,112 @@
+     */
+    private static final int NUMBER_OF_SOUNDS = 2;
+
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 25;
+
+    /**
+     * Chance to play a basic sound.
+     */
+    private static final int BASIC_SOUND_CHANCE = 100;
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private FishermanSounds()
+    {
+        /*
+         * Intentionally left empty.
+         */
+    }
+
+    /**
+     * Containing the female fisherman sounds.
+     */
+    public static final class Female
+    {
+        public static final SoundEvent generalPhrases = ModSoundEvents.registerSound("mob.fisherman.female.generalPhrases");
+        public static final SoundEvent noises = ModSoundEvents.registerSound("mob.fisherman.female.noise");
+        public static final SoundEvent iGotOne = ModSoundEvents.registerSound("mob.fisherman.female.iGotOne");
+        public static final SoundEvent needFishingRod = ModSoundEvents.registerSound("mob.fisherman.female.needFishingRod");
+        public static final SoundEvent offToBed = ModSoundEvents.registerSound("mob.fisherman.female.offToBed");
+        public static final SoundEvent badWeather = ModSoundEvents.registerSound("mob.fisherman.female.badWeather");
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Female()
+        {
+            /*
+              Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * Containing the male fisherman sounds.
+     */
+    public static final class Male
+    {
+        public static final SoundEvent generalPhrases = ModSoundEvents.registerSound("mob.fisherman.male.generalPhrases");
+        public static final SoundEvent noises = ModSoundEvents.registerSound("mob.fisherman.male.noise");
+        public static final SoundEvent iGotOne = ModSoundEvents.registerSound("mob.fisherman.male.iGotOne");
+        public static final SoundEvent needFishingRod = ModSoundEvents.registerSound("mob.fisherman.male.needFishingRod");
+        public static final SoundEvent offToBed = ModSoundEvents.registerSound("mob.fisherman.male.offToBed");
+        public static final SoundEvent badWeather = ModSoundEvents.registerSound("mob.fisherman.male.badWeather");
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Male()
+        {
+            /*
+             * Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * Plays fisherman sounds.
+     * @param worldIn the world to play the sound in.
+     * @param position the position to play the sound at.
+     * @param isFemale the gender.
+     */
+    public static void playFishermanSound(World worldIn, BlockPos position, boolean isFemale)
+    {
+        //Leaving it as switch-case we may add further random sound categories here (Whistling, singing, etc).
+        switch (rand.nextInt(NUMBER_OF_SOUNDS + 1))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement by "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1301)